test/test_rtlog.cpp

```diff
@@ -191,52 +203,43 @@ TEST(RtlogTest, ErrorsReturnedFromLog) {
     va_end(args);
 
     EXPECT_STREQ(buffer.data(), "Hello, 12");
-    EXPECT_EQ(strlen(buffer.data()), maxMessageLength - 1);
+    EXPECT_EQ(strlen(buffer.data()), this->maxMessageLength - 1);
   };
-  EXPECT_EQ(truncatedLogger.PrintAndClearLogQueue(InspectLogMessage), 1);
+  EXPECT_EQ(this->logger.PrintAndClearLogQueue(InspectLogMessage), 2);
```

Comment:
Question for future me to figure out: Why did this need to change to 2? 

More specifically, why did it return 1 in the first place? It should have enqueued both the truncated and non truncated messages (2). This indicates that there was a bug in the original version.

Need to investigate more before possibly changing some behavior.  I think this new way is correct, I just want to understand more